docs/spec/v1/gitrepositories.md

```diff
@@ -104,7 +104,7 @@ Git repository.
 **Note:** Unlike using `git`, the
 [shorter scp-like syntax](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_the_ssh_protocol)
 is not supported for SSH addresses (e.g. `user@example.com:repository.git`).
-Instead, the valid URL format is `ssh://user@example.com:22/repository.git`.
+Instead, the valid URL format is `ssh://user@example.com/repository.git` (only 1 `:` allowed).
```

Comment:
I do not feel this really adds anything, if not actually making this harder to understand.

Reply:
Fair enough